.ci.yaml

```diff
@@ -3236,7 +3235,6 @@ targets:
       validation_name: Verify x64 binaries codesigned
 
   - name: Mac_arm64 verify_binaries_codesigned
```

Comment:
@XilaiZhang do you expect this to pass now after https://github.com/christopherfujino/codesign.py/pull/5?

Reply:
Probably not. The error we have been seeing comes from some weird luci git behavior , and not codesign logic. The git error would look something like (e.g. https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8790347300087503201/+/u/Verify_binaries_codesigned/verify_binaries_codesigned/test_stderr) Previously we were attempting to resolve it with Christopher but it wasn't trivial.

Reply:
I mean in post-submit, not the git tree error. Won't the arm binaries be codesigned going forward on https://github.com/christopherfujino/codesign.py/pull/5?

Reply:
right the binaries would be codesigned.

I could be wrong, but my understanding is that the check would still fail because of how the test is written. Like the implementation of verify_binary_codesigned would invoke a git logic that fails. 
i think the check might fail not because of the artifact being tested, but because of the test logic itself.

Yeah when engine v2 is ready we would not need this test anymore.

Reply:
You're saying https://github.com/flutter/flutter/issues/116611 fails in post-submit as well?  My understanding of that error is that it's trying to check out the SHA on my fork, which doesn't exist in flutter/flutter until after it's merged.  So in post-submit it should get past that part and succeed if the binaries are signed.

Reply:
Ahhh Jenn is right ! !
yeah i agree. post submit sounds good!
Yeah i just briefly looked at https://flutter-dashboard.appspot.com/#/build?taskFilter&authorFilter&messageFilter&hashFilter&showMac=true&showWindows=true&showiOS=true&showLinux=true&showAndroid=true&showStaging=false&repo=flutter&branch=flutter-3.7-candidate.1 and all the verify mac cosesign tests are yellow.
But i think Jenn is right, they are queued probably for infra reasons and not codesign logics. maybe @ricardoamador  would know why the checks are all queued in yellow? Thank you!

Reply:
Hm, it is running in some form but as `Mac stable verify_binaries_codesigned`, I don't know where that's defined though...

https://ci.chromium.org/p/flutter/builders/prod/Mac%20stable%20verify_binaries_codesigned

I'll file a new issue about this.